cvcl/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+class User(models.Model)
+    username = models.CharField(max_length=30)
+    password = models.CharField(max_length=30)
+
+    def __str__(self):
+    	return self.username
+
+class User_group(models.Model):
```

Comment:
Should be `UserGroup`
Read: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style